VK/VK/Services/Models/FeedModels/FeedModels.swift

```diff
@@ -0,0 +1,109 @@
+// FeedModels.swift
```

Comment:
```suggestion
// FeedModels.swift
```
вроде как одна структура -- один файл?)